src/components/FilterButton.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+function FilterButton (props) {
+    return (
+        <div className="filter-button">
+            <label for="free-movies">Free Movies </label>
```

Comment:
this file should not show up in code review as it was previously reviewed and merged in `development` branch.

Change the PR from master to development as we want this feature to be merged in development first according to `gitflow`.
https://datasift.github.io/gitflow/IntroducingGitFlow.html